spec/models/sponsor_spec.rb

```diff
@@ -357,6 +358,82 @@
       specify '.all_inactive should return sponsors with Inactive status only' do
         expect(Sponsor.all_inactive).to match_array [inactive_sponsor]
       end
+
+      describe '.filter' do
+        before :each do
+          @filter_params = {
+            name_option: "",
+            name_value: nil,
+            gender: sponsor.gender,
+            branch: sponsor.branch_id,
+            organization: sponsor.organization_id,
+            status: sponsor.status_id,
+            sponsor_type: sponsor.sponsor_type_id,
+            agent: sponsor.agent_id,
+            city: sponsor.city,
+            country: sponsor.country,
+            created_at_from: sponsor.created_at - 1.day,
+            created_at_until: sponsor.created_at + 1.day,
+            updated_at_from: sponsor.updated_at - 1.day,
+            updated_at_until: sponsor.updated_at + 1.day,
+            start_date_from: sponsor.start_date - 1.day,
+            start_date_until: sponsor.start_date + 1.day,
+            request_fulfilled: sponsor.request_fulfilled,
+            active_sponsorship_count_option: "",
+            active_sponsorship_count_value: nil
+          }
```

Comment:
I am still not sure where to put this. It is a hash that is used in multiple spec files. Can't use FactoryGirl because it is not a model.
Any suggestions are welcome.